pynest/setup.py.in

```diff
@@ -19,20 +19,47 @@
 # You should have received a copy of the GNU General Public License
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
 
-from distutils.core import setup
+from setuptools import setup
 
 setup(
-    name='PyNEST',
+    name='nest',
     version='@NEST_VERSION_STRING@',
-    description='PyNEST provides Python bindings for NEST',
+    description='Python bindings for NEST',
     author='The NEST Initiative',
     url='https://www.nest-simulator.org',
     license='GPLv2+',
     packages=['nest', 'nest.tests', 'nest.tests.test_sp',
               'nest.tests.test_spatial', 'nest.lib'],
-    package_dir={'nest': '@PROJECT_SOURCE_DIR@/pynest/nest'},
-    package_data={
-        'nest': ['pynest-init.sli'],
-        'nest.tests': ['test_aeif_data_lsodar.dat'],
+    install_requires=['numpy', 'scipy'],
+    extras_require={
+        'test': ['nose', 'matplotlib']
+    },
+    classifiers=[
+        'Development Status :: 6 - Mature',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Intended Audience :: Science/Research',
+        'Topic :: Scientific/Engineering',
+        'Topic :: Scientific/Engineering :: Artificial Intelligence',
+
+    ],
+    python_requires='>=3.8, <4',
```

Comment:
I don't think many do, but I think everyone should until the release candidates are announced and it can be checked whether NEST works under it or not. It's safer than hoping someone will remember to add this when Python 4 comes around

Reply:
OK.